perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -54,6 +80,107 @@
                   'num_machines': 1}
 
 
+JOB_FILE_TEMPLATE = """
```

Comment:
You're right, and the results are not comparable to the results from the old benchmark. We don't want it to be too easy to confuse the two benchmarks. I'm not sure how to handle this properly. I think the choices are either put something in the metadata to indicate what the benchmark is, make my code a new benchmark, or maybe have fio_benchmark record samples with multiple test names (I was thinking that each rw mode could be its own test, so "read", "randread", "write", and "randwrite" would all be different). I can argue for each of these options, but I can't convince myself that one of them is clearly right.